deploy/example/vpa/vertical-pod-autoscaler.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
```

Comment:
pls add this yaml config verification in hack/verify-yamllint.sh script

Reply:
done